Formula/ntopng.rb

```diff
@@ -56,14 +52,20 @@ class Ntopng < Formula
 
   fails_with gcc: "5"
 
+  # Allow dynamic linking with nDPI
+  patch :DATA
```

Comment:
Has this patch been upstreamed?

Reply:
There were 2 attempts of it:
- https://github.com/ntop/ntopng/issues/3379
- https://github.com/ntop/ntopng/pull/6318

Upstream wants to keep static linkage by default (and allow to make it dynamic via additional parameter) PRs authors didn't have time to do it (yet?).